products/pluto/recipe-minor.yml

```diff
@@ -3,6 +3,8 @@ base_recipe: recipe.yml
 version_type: minor
 version_strategy: bump_latest_release
 product: db-pluto
+vars:
+  VERSION_TYPE: minor
```

Comment:
since `version_type` is already declared above this, I think it can just be added to `recipe.vars` during `plan_recipe` [here](https://github.com/NYCPlanning/data-engineering/blob/main/dcpy/lifecycle/builds/plan.py#L84) so that it becomes an envar [here](https://github.com/NYCPlanning/data-engineering/blob/main/dcpy/lifecycle/builds/plan.py#L105)

maybe something like:
```python
if recipe.version_type is not None: recipe.vars["VERSION_TYPE"] = recipe.version_type
```

Reply:
wow. I didn't see this var being already declared. Thanks!